test/jacobian.jl

```diff
@@ -0,0 +1,16 @@
+include("utils.jl")
+
+@testset "jacobian" begin
+    for T in (Float64, ComplexF64)
+        A = sprand(T, 10, 5, 0.2)
+        b = randn(T, 5)
+        JFb = forwarddiff_mm_jacobian(A, b)
+        JNb = nilang_mm_jacobian(A, b)
```

Comment:
If I make it correctly, do we only check the jacobian for matrix-matrix multiplication here? We might also want to check matrix-adjoint multiplication.

Reply:
Either remove the gradient related things completely or write tests for them would work. As we discussed yesterday, we prefer removing it first.